src/formatter.py

```diff
@@ -0,0 +1,125 @@
+
+
+class DictFormatter:
```

Comment:
Do we need exactly a `dict` converter?
I mean, some game components can be conveniently represented as lists or other primitives...